### DISCUSSION TITLE
mergify removing reviews

### DISCUSSION THREAD

**Discussion by ansiwen** [ceph/go-ceph#445] [General]
Mergify is removing reviews, even if the diff didn't change at all (rebases with zero code changes in the diff contexts), which is quite annoying. Especially when there are several PRs waiting. It doesn't even make sense that I reapprove all of @phlogistonjohn 's PRs, because as soon as one is merged they will disappear again. Can we tune that? If not, I would even rather prefer to disable this feature.

**Comment by phlogistonjohn** [comment]
FWIW, github has just added a built in automerge feature (in beta). I have very little experience with it, but from what I have it's a bit quirky too. But it could be less annoying. Perhaps after the release we should experiment with that?

**Comment by anoopcs9** [comment]
IIUC, this is now implemented within current mergify rules.
